libs/content/workspace/src/tab/svg_editor/pen.rs

```diff
@@ -151,10 +112,10 @@ impl Pen {
                         }
                     }
 
-                    if self.detect_snap(&p.data, payload.pos, pen_ctx.buffer.master_transform) {
-                        self.end_path(pen_ctx, true);
-                        return;
-                    }
+                    // if self.detect_snap(&p.data, payload.pos, pen_ctx.buffer.master_transform) {
+                    //     self.end_path(pen_ctx, true);
+                    //     return;
+                    // }
```

Comment:
?

Reply:
snap is broken and i'm not interested in supporting it until the pen tool is in a stable place which is after i add predicted/coeleased touch support and native svg thickness. the comment serve as a note to myself, I've put a todo comment to make it more clear